src/Tasks/Common/Resources/Strings.resx

```diff
@@ -690,4 +690,8 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1146: PackAsTool does not support TargetPlatformIdentifier being set. For example, TargetFramework cannot be net5.0-windows, only net5.0. PackAsTool also does not support UseWPF or UseWindowsForms when targeting .NET 5 and higher.</value>
     <comment>{StrBegin="NETSDK1146: "}</comment>
   </data>
+  <data name="WorkloadNotInstalled" xml:space="preserve">
+    <value>NETSDK1147: The following workload packs were not installed: {0}</value>
```

Comment:
@dsplaisted I incremented your error numbers by 1 as it conflicted with Will's change.  Let me know if there are any concerns (like documentation or tests looking for that number).